server/src/test/kotlin/org/javacs/kt/LintTest.kt

```diff
@@ -26,4 +30,27 @@ class LintTest : SingleFileTestFixture("lint", "LintErrors.kt") {
         assertThat(diagnostics, not(empty<Diagnostic>()))
         assertThat(languageServer.textDocumentService.lintCount, lessThan(5))
     }
+
+    @Test fun `lintting should not be dropped if another lintting is running`() {
```

Comment:
```suggestion
    @Test fun `linting should not be dropped if another linting is running`() {
```

Linting should be spelled with a single t ([see here](https://en.wiktionary.org/wiki/linting)).